tests/CellGridTranslatorTest.java

```diff
@@ -0,0 +1,52 @@
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CellGridTranslatorTest {
+
+    @Test
+    public void whenGivenA2dArrayOfCellsReturnsA2dArrayOfStrings(){
+        CellGrid cellGrid = new CellGrid(3,3);
+
+        String[][] expectedOutput = {{"x", "x", "x"}, {"x", "x", "x"}, {"x", "x", "x"}};
+        String[][] actualOutput = CellGridTranslator.getCellGridAsStringArray(cellGrid);
+
+        assertArrayEquals(expectedOutput, actualOutput);
+
+    }
+
+    @Test
+    public void whenACellsIsAliveStringIsSetToAnO(){
+        CellGrid cellGrid = new CellGrid(3,3);
+
+        cellGrid.setCellToAlive(1,1);
+
+        String[][] expectedOutput = {{"x", "x", "x"}, {"x", "o", "x"}, {"x", "x", "x"}};
+        String[][] actualOutput = CellGridTranslator.getCellGridAsStringArray(cellGrid);
+
+        assertArrayEquals(expectedOutput, actualOutput);
+
+
+    }
+
+    @Test
+    public void whenACellIsDeadStringIsSetToAnX(){
+        CellGrid cellGrid = new CellGrid(3,3);
+
+        cellGrid.setCellToAlive(0,0);
+        cellGrid.setCellToAlive(0,1);
+        cellGrid.setCellToAlive(0,2);
+        cellGrid.setCellToAlive(1,0);
+        cellGrid.setCellToAlive(1,1);
+        cellGrid.setCellToAlive(1,2);
+        cellGrid.setCellToAlive(2,0);
+        cellGrid.setCellToAlive(2,1);
+
+        String[][] expectedOutput = {{"o", "o", "o"}, {"o", "o", "o"}, {"o", "o", "x"}};
+        String[][] actualOutput = CellGridTranslator.getCellGridAsStringArray(cellGrid);
+
+        assertArrayEquals(expectedOutput, actualOutput);
+
+    }
+
```

Comment:
This is interesting, and I guess I have a similar question.
The gist is, you're test name is pointing at a particular test case, and it's an important test case.
But maybe we should compare that to what we're asserting on. Mostly thinking about the level of granularity in the assertion. 
This is still correct, but ends up being a little more heavyweight as a test than what we probably require for this case.

Reply:
yeah, i feel like these tests are not quite right. i've also broken it for the time being lol